### ISSUE TITLE
jupyter-js-widgets sub-package install throws error

### ISSUE THREAD

**Issue by fperez** [ipython/ipywidgets#582]
I'm getting JS errors after pulling master that I'm not sure how to debug, so following how @SylvainCorlay helped me recently, I tried to just re-install the js package, but I'm getting this error:

```
(master)denali[jupyter-js-widgets]> npm install --save jupyter-js-widgets
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/node/6.0.0/bin/node" "/usr/local/bin/npm" "install" "--save" "jupyter-js-widgets"
npm ERR! node v6.0.0
npm ERR! npm  v3.8.6
npm ERR! code ENOSELF

npm ERR! Refusing to install jupyter-js-widgets as a dependency of itself
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/fperez/dev/ipython/ipywidgets/jupyter-js-widgets/npm-debug.log
```

I've put the debug log [here](https://gist.github.com/fperez/5b511f555cb3eb781663760e47dab9c4) in case it helps.

**Comment by SylvainCorlay** [comment]
You need to remove the `--save jupyter-js-widgets` from the command.

**Comment by minrk** [comment]
@SylvainCorlay readme says the command is right, but the dir is wrong:

    npm install --save jupyter-js-widgets

should be run at the root of the widgets repo. Is that right?

**Comment by minrk** [comment]
I think this was fixed by #583
